### ISSUE TITLE
Can not get data from ATT&CK workbench

### ISSUE THREAD

**Issue by dickens88** [mitre-attack/attack-navigator#324]
Hi i'm working with ATT&CK Navigator(v4.3) and Workbench(v1.0.1). Currently i have deployed a ATT&CK Workbench, i'm trying to integrate Navigator by changing the `/src/assets/config.json`. the config looks like this. 

```
{
    "versions": [
        {
            "name": "ATT&CK Workbench Data", 
            "domains": [
                {   
                    "name": "Enterprise", 
                    "data": ["http://localhost:3000/api/stix-bundles/?domain=enterprise-attack"]
                },
                {   
                    "name": "Mobile", 
                    "data": ["http://localhost:3000/api/stix-bundles/?domain=mobile-attack"]
                },
                {
                    "name": "ICS",
                    "data": ["http://localhost:3000/api/stix-bundles/?domain=ics-attack"]
                }
            ]
        }
    ]
}
```
after this i clicked create new Layer in Navigator, i can see the config related to Workbench, but after i click `create` it just rolling and rolling without show any notice or result..

![image](https://user-images.githubusercontent.com/8390209/126278700-eb280cea-cc65-4991-adc9-5193985e2daf.png)

is there any knowns issues relate to the integration of Workbench?

**Comment by isaisabel** [comment]
Hi @dickens88,

Based on the screenshot you added in the parallel issue [here](https://github.com/center-for-threat-informed-defense/attack-workbench-frontend/issues/230), I'm guessing you created a new technique/tactic/group/software/mitigation but didn't assign it an ATT&CK ID. 

That ATT&CK Navigator doesn't properly handle that is definitely a bug, but until we fix this on the Navigator repo the solution will be to assign the object an ATT&CK ID.

The fix for Navigator will likely be to filter out objects with no assigned ID. This is how we typically do it with other ATT&CK software -- objects with no ID (excepting those that don't normally have ATT&CK IDs such as relationships) are considered "drafts" and are filtered out in applications consuming the STIX.

So either way, adding an ID will be required to view your creation in this application.

**Comment by dickens88** [comment]
@isaisabel   Like you said, I did forget to write the group ID, when I deleted those records, the problem was solved. Thank you so much for your help. This tool is awesome, I believe it will be very useful to my organization. :)

**Comment by dickens88** [comment]
In addition, maybe you can make the ID as a required item in the form, if it really affects the processing of the application...

**Comment by isaisabel** [comment]
Glad that fixed it and that you like the tool!

I've opened an issue for warning the user if no ID has been specified (https://github.com/center-for-threat-informed-defense/attack-workbench-frontend/issues/231). I do think there are use cases for having objects without IDs (e.g the drafts case I mentioned above) but we definitely do need to make the ramifications of omitting an ID more apparent.

**Comment by isaisabel** [comment]
The improved warning for missing ATT&CK IDs has been implemented on the develop branch and will be released to master with the release of v4.4.
